packages/salesforcedx-vscode-apex-debugger/src/messages/i18n.ts

```diff
@@ -16,8 +16,11 @@
  * If ommitted, we will assume _message.
  */
 export const messages = {
+  config_name_text: 'Launch Apex Debugger',
   select_exception_text: 'Select an exception',
   select_break_option_text: 'Select break option',
   always_break_text: 'Always break',
-  never_break_text: 'Never break'
+  never_break_text: 'Never break',
+  language_client_not_ready:
+    'Unable to retrieve breakpoint info from language server, language server is not ready'
```

Comment:
Can we provide more guidance here? How long should the user wait before trying again? Maybe we can say something like, `'Unable to retrieve breakpoint info from language server. The language server isn’t ready yet. Try again in a few minutes.'`.

Reply:
@lcampos

Reply:
This is challenging because the time the language server takes to load depends on the size of the project the customer is using.